### ISSUE TITLE
verify_mode not set in puma. 

### ISSUE THREAD

**Issue by pretoritus** [logstash-plugins/logstash-input-http#46]
We get following error on stdout when we start Logstash with http input configured:

```
Listen loop error: #<TypeError: can't convert nil into Integer>
org/jruby/puma/MiniSSL.java:162:in `initialize'
org/jruby/puma/MiniSSL.java:131:in `server'
/opt/software/logstash/product/logstash-2.2.2/vendor/bundle/jruby/1.9/gems/puma-2.16.0-java/lib/puma/minissl.rb:167:in `accept_nonblock'
/opt/software/logstash/product/logstash-2.2.2/vendor/bundle/jruby/1.9/gems/puma-2.16.0-java/lib/puma/server.rb:328:in `handle_servers'
org/jruby/RubyArray.java:1613:in `each'
```

It seems that "verify_mode" in puma is not set which is per default `nil`. 

**org/jruby/puma/MiniSSL.java:162:**
`long verify_mode = miniSSLContext.callMethod(threadContext, "verify_mode").convertToInteger().getLongValue();`

This affects the Versions 2.1.3, 2.2.1 and 2.2.2.  Version 2.2.0 has not this issue.

Release notes of puma 2.16.

=== 2.16.0 / 2016-01-27

* 7 minor features:

  * Add 'set_remote_address' config option
  * Allow to run puma in silent mode
  * Expose cli options in DSL
  * Support passing JRuby keystore info in ssl_bind DSL
  * Allow umask for unix:/// style control urls
  * Expose `old_worker_count` in stats url
  * **Support TLS client auth (verify_mode) in jruby**

**Comment by pretoritus** [comment]
Can i support closing this ticket?

**Comment by jsvd** [comment]
closed by #37
